coalib/parsing/Globbing.py

```diff
@@ -29,6 +29,24 @@ def _end_of_set_index(string, start_index):
     return closing_index
 
 
+def glob_escape(string):
+    """
+    Escapes the given string with `[c]` pattern.
+
+    :param string: String that is to be escaped with `[ ]`.
+    :return:       Escaped string in which all the special glob characters
+                   `()[]|?*` are escaped.
+    """
+    output = ""
+    special_chars = "()[]|?*"
```

Comment:
Agreed, there may be a `unescape_glob` in the future which might want the sequence again and `Globbing.SPECIAL_CHARS` will come in handy.